letsencrypt/plugins/webroot.py

```diff
@@ -1,4 +1,43 @@
-"""Webroot plugin."""
+"""Webroot plugin.
+
+Content-Type
+------------
+
+This plugin requires your webserver to use a specific `Content-Type`
+header in the HTTP response.
+
+Apache2
+~~~~~~~
+
+.. note:: Instructions written and tested for Debian Jessie. Other
```

Comment:
You could probably go for more abstraction.  Instead of a2enmod (which is Debian-specific), say "enable the headers module".  Then instead of naming the conf-available dir, say, "make this is applied".

For extra fanciness, print the Debian-specific help when `/etc/debian_version` exists, and a more generic version otherwise :)